src/sandbox/runtime/actions/edit.js

```diff
@@ -55,3 +70,20 @@ export const clearRange = undoable((thisSheet, range) => {
   const found = query(thisSheet.get().blocks).getInRange(normalize(range));
   for (let blockId of found) clearBlock(thisSheet, blockId);
 });
+
+export const commentRange = undoable((thisSheet, range, cb) => {
+  const blocks = thisSheet.get("blocks");
+  const found = query(blocks).getInRange(normalize(range));
+
+  for (let blockId of found) {
+    const block = blocks.get(blockId);
+    const [row, col] = block.position;
+    const value = block
```

Comment:
If block is undefined block.position would have thrown the line before

Reply:
Could you please provide an example?

Reply:
OK. Fixed!

Reply:
How is this fixed??? Renamed the callback??? DRY should not mean spaghetti!
toggleComment is just a utility function that you need both in edit.js and in the component. Then wtf is the reason for it to remain in the component???